build/deploy/README.md

```diff
@@ -1,99 +1,14 @@
 # Kubernetes deployment via Tanka
 
-This folder contains a set of configuration files to be used by
-[tanka](https://tanka.dev/install) to deploy a single DSS instance via
-Kubernetes following the procedures found in the [build](..) folder.
+The documentation and configuration have been moved to [deploy/services](../../deploy/services/tanka).
 
-## Architecture
+## Migrating configurations to new location
```

Comment:
Let's add a note in the PR description pointing people to this page & section who might be reading through the change log to see what they need to do to move to the newest version with their system